.github/copilot-instructions.md

```diff
@@ -0,0 +1,34 @@
+# GitHub Copilot Instructions
+
+## Project Overview
+
+MPDX is a Next.js 15 React application built with TypeScript using pages router.
+
+## Technology Stack
+
+- **Frontend**: Next.js v15, React, Material UI v5, TypeScript
+- **Data Loading**: Apollo Client GraphQL
+- **Testing**: Jest, React Testing Library v13
+- **Internationalization**: react-i18next
+- **Code Quality**: ESLint, Prettier
```

Comment:
Should you also add a part about forms and that it should use Formik and yup for validation